gensim/models/ldamodel.py

```diff
@@ -1047,6 +1047,12 @@ def load(cls, fname, *args, **kwargs):
         """
         kwargs['mmap'] = kwargs.get('mmap', None)
         result = super(LdaModel, cls).load(fname, *args, **kwargs)
+
+        random_state_fname = utils.smart_extension(fname, '.random_state')
+        try:
+            result.random_state = super(LdaModel, cls).load(random_state_fname, *args, **kwargs)
+        except Exception as e:
+            logging.warning("failed to load random_state from %s: %s", random_state_fname, e)
```

Comment:
Have you tested it with 0.13.1 version? 
the fix is to assign `get_random_state(None)` if it is missing on disk